### ISSUE TITLE
Useless Overdrive 5-7 calls for AMD RX 6000

### ISSUE THREAD

**Issue by zeule** [LibreHardwareMonitor/LibreHardwareMonitor#477]
When `ADL_Overdrive_Caps()` reports overdrive version 8, `AmdGpu.Update()` first tries all overdrive versions from 5 to N, and all the calls fail (at least for 6900 xt).

**Comment by PhyxionNL** [comment]
Correct, but sadly this seems necessary to get full detection across all drivers and devices as ADL is really unreliable.

**Comment by zeule** [comment]
But is it really required to do that on each update?

**Comment by PhyxionNL** [comment]
Probably not but not all calls seem to be reliable all of the time, so maybe it can stop calling it after something like 30 failed calls (and reset to 0 when successful). It should probably also reset after system sleep/hibernate.
